onyo/lib/onyo.py

```diff
@@ -126,7 +126,7 @@ def _get_files(self) -> set[Path]:
         Return a set of all files in the repository (except under .git).
         """
         log.debug('Acquiring list of files')
-        files = {Path(x) for x in self._git(['ls-files']).splitlines() if x}
+        files = {Path(x) for x in self._git(['ls-files', '-z']).split('\0') if x}
```

Comment:
Doesn't this fix also need to be applied to the other `_get_files_*()` functions?